packages/ra-ui-materialui/src/detail/TabbedShowLayoutTabs.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component, Children, cloneElement } from "react";
+import PropTypes from "prop-types";
+import Tabs from "@material-ui/core/Tabs";
+import Divider from "@material-ui/core/Divider";
+import { withRouter, Route } from "react-router-dom";
```

Comment:
I suspect that some of these are unused, please check the linter.

Reply:
When I run `make linter` it's only returning some `console.log` calls in other files (ListGuesser for example). When running `make prettier`, it changes most of the projects files with a different style. Did I set up something wrong?

Reply:
We currently have a problem with the `make prettier` command, so you can ignore it. I'm surprised that `make lint` doesn't show the unused imports. You'll have to check those imports one by one.

Reply:
Done in https://github.com/marmelab/react-admin/pull/2740/commits/c2dd91d80f392fe0480745a0b2f9218d42f0fb26